src/collective/importexport/browser/import_view.py

```diff
@@ -258,16 +274,61 @@ def export_file(result, header_mapping, request=None):
                     continue
                 if value is field.missing_value:
                     continue
+                if IRichTextValue.providedBy(value):
+                    # Convert to plain text
+                    transforms = api.portal.get_tool('portal_transforms')
+                    value = transforms.convertTo(
+                                target_mimetype='text/plain',
+                                orig=value.raw_encoded,
+                                encoding=value.encoding,
+                                mimetype=value.mimeType)
+                    value = safe_unicode(value.getData())
+                    break
+
+                if IList.providedBy(field):
+                    # Check if items in list point to objects and use titles
+                    # instead
+                    names=[]
+                    for v in value:
```

Comment:
Maybe use a better name than `v`. Is this an item uuid, or id inside of a folder? In that case `item_uuid` or `item_id` would be more readable.